__tests__/views/selection.js

```diff
@@ -23,7 +23,7 @@ describe('Selection view', function () {
         }).should.throw(RangeError);
 
         (function () {
-            m.selectionView([1, 1, 2]);
+            m.selectionView([1, 1]);
```

Comment:
that's for the test of `TypeError`, I splited the implementation of checkIndex, by rows and columns, therefore it throwed first the `RangeError` than the `TypeError`